frontend/src/metabase/query_builder/components/SidebarContent/SidebarContent.tsx

```diff
@@ -41,7 +41,7 @@ function SidebarContent({
       <SidebarContentMain data-testid="sidebar-content">
         {(title || icon || onBack) && (
           <SidebarHeader
-            className="mx3 my2 pt1"
+            className="mx3 my2 pt1 sidebar-header"
```

Comment:
Think we can remove the `sidebar-header` class since the styling is now moved to emotion?